### DISCUSSION TITLE
load() function is returning undefined

### DISCUSSION THREAD

**Discussion by omidamraei** [adonisjs/core#2736] [Help] [answered]
A simple walk-through:

    const user = await User.find(1);

    // files are returned
    const files = await user.related('files').query()

    // `files` is undefined
    const files = await user.load('files')

Adonis version: 5.1.7
Lucid version: 14.1.0

**Comment by omidamraei** [comment, marked as answer]
I found out what the problem is. The `load` function is not working that way. 
But this is working:

```
await user.load('files')
user.files
```

**Reply by sasha172017** [reply]
await User.query().preload('files') //preload
user.files
